shared/fixes/bash/rpm_verify_hashes.sh

```diff
@@ -0,0 +1,12 @@
+# platform = multi_platform_rhel, multi_platform_fedora
+
+# Find which files have incorrect hash (not in /etc, because there are all system related config. files) and then get files names
+files_with_incorrect_hash="$(rpm -Va | grep -E '^..5.* /(bin|sbin|lib|lib64|usr)/' | sed -r 's;^.*\s(.+)$;\1;g' )"
```

Comment:
This script should probably be similar to `shared/fixes/bash/rpm_verify_permissions.sh`

Also, we shouldn't limit to specific directories.

Reply:
Would it make sense to update OVAL definition too?
https://github.com/OpenSCAP/scap-security-guide/blob/master/shared/checks/oval/rpm_verify_hashes.xml#L33

Reply:
Note that it was deliberate change, to limit the directories in the oval. Are we running in circles?

Reply:
@dahaic no you are right. I am the one running in circles.